### DISCUSSION TITLE
Fseq file upload

### DISCUSSION THREAD

**Discussion by surajchauhan6631** [forkineye/ESPixelStick#740] [Q&A]
I want to use espixelstick to work in standalone mode and cycle through the fseq sequence. I want to upload a fseq file from vixen software in which there is no option of fpp connect. Can I upload files manually to the sd card ?

**Comment by MartinMueller2003** [comment]
With Vixen you use export to create your fseq files in uncompressed mode. Then you drag & drop the files onto the file download are in the file manager page of the PixelStick UI.

Create a playlist (definition is in the readme) and select the playlist in the file to play on the device configuration screen. That list will be played until you stop it.

**Reply by surajchauhan6631** [reply]
Also want to know can espixelstick be used to  get 4 dmx output port handling 4 dmx universe, one universe on each port. Or can we also get 8 dmx port output.

**Reply by surajchauhan6631** [reply]
what export setting in vixen we have to use for exporting fseq files which can be used with espixelstick?. I have many files in which only one  files is running that sequence. while when i upload other fseq file it doesn't run that file. and only show time Elapsed and time remaining to be 0.00

**Reply by MartinMueller2003** [reply]
You can find out what the problem is by going into advanced mode on the admin screen and then returning to the status screen. there will be an additional line with the last status encountered by the player.
On the setup screen, make sure you have a dedicated controller set up for the ESP. I am going to call it ESP1 for this discussion.
On the export screen

- format: 2.6+
- select only ESP1 in the list of controllers.
- set the frame rate to whatever you are using. ESP V4 supports up to 40 fps (25ms)
- set the compression to uncompressed
- press start
When finished, copy the file to the ESP.

**Reply by surajchauhan6631** [reply]
where i have to select esp1 in list of controllers
![Screenshot (40)](https://github.com/forkineye/ESPixelStick/assets/35253321/e8e930b9-0517-4f5d-9f45-2437d6f2d89f)

**Reply by surajchauhan6631** [reply]
and one more issue i am facing is that i am lost the output1 settings for dmx  even after clicking the save changes button.

**Reply by MartinMueller2003** [reply]
ESP1 is a made up name. You have only one controller defined so you do not have an issue.

**Reply by surajchauhan6631** [reply]
and one more issue i am facing is that i am lost the output1 settings for dmx even after clicking the save changes button. so i have to change the setting again. why this issue is persisting.

**Reply by MartinMueller2003** [reply]
I have never seen such a thing. Can you send me a log of the output when this is happening?

**Comment by MartinMueller2003** [comment]
ESPixelStick V4 support up to 8 serial outputs. You would need to create a base card that has the GPIO to RS-485 conversion or you could use 8 ttl to rs485 conversion boards.

**Reply by surajchauhan6631** [reply]
Ok, so using a customize board I can receive 8 artnet universe and send one universe on each dmx output port of 8 ports

**Comment by MartinMueller2003** [comment]
or E1.31 or DDP or fseq file on a local SD.

**Comment by MartinMueller2003** [comment]
I have never seen such a thing. Can you send me a log of the output when this is happening?

**Reply by surajchauhan6631** [reply]
I want to create custom board for Espixelstick with 8 port pixel output with ethernet connectivity for Artnet or e1.31 data input. Can you guide me which esp32 pin should be connected to which pin of ethernet chip.

**Reply by surajchauhan6631** [reply]
??

**Reply by MartinMueller2003** [reply]
ALL ESP32 devices are required to use the same pins. You can see an example of an 8 port controller by looking at the QuinLed Octa hardware platform definitions.
